scripts/linters/python_linter.py

```diff
@@ -244,9 +282,11 @@ def perform_all_lint_checks(self):
             return []
 
         import_order_check_message = self._check_import_order()
+        test_function_check_message = self._check_non_test_files()
         job_registry_check_message = (
             self._check_that_all_jobs_are_listed_in_the_job_registry_file())
-        return import_order_check_message + job_registry_check_message
+        return (import_order_check_message + job_registry_check_message +
```

Comment:
assign these messages to a variable named `all_messages` and then return that variable.

Reply:
done